css/index.css

```diff
@@ -1,8 +1,11 @@
-@import url("https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.css");
+@import url(https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.css);
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300);
+@import url(http://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css);
```

Comment:
Loading through ```<link>``` asynchronously loads all of the stylesheets. Loading through ```@import``` requires the browser to sequentially load each stylesheet before it can load all of the others.